addons/mail/tests/test_mail_render.py

```diff
@@ -592,7 +592,6 @@ def patched_mail_template_default_values(model):
         }))
         template = template.save()
         self.assertFalse(template.lang)
-        self.assertFalse(template.email_cc)
```

Comment:
Why is `template.lang` okay in both cases tho? As they are both in the same hidden panel.

Reply:
:chicken: Le ?